management/src/main/resources/reference.conf

```diff
@@ -16,6 +16,45 @@ akka.management {
     # The value will need to be from 0 to 65535.
     port = 19999
 
+    # Use this setting to bind a network interface to a different hostname or ip
+    # than the HTTP Server for Http Cluster Management.
+    # Use "0.0.0.0" to bind to all interfaces.
+    # akka.management.http.hostname if empty
+    bind-hostname = ""
+
+    # Use this setting to bind a network interface to a different port
+    # than the HTTP Server for Http Cluster Management. This may be used
+    # when running akka nodes in a separated networks (under NATs or docker containers).
+    # Use 0 if you want a random available port. Examples:
+    #
+    # akka.management.http.port = 2552
```

Comment:
bad idea to show this number in the example though? That's the usual remoting port

Reply:
Fixed